proposed/cache.md

```diff
@@ -348,6 +359,17 @@ interface CacheItemPoolInterface
     public function save(CacheItemInterface $item);
 
     /**
+     * Persists multiple cache items immediately.
+     *
+     * @param array|\Traversable $items
+     *   A traversable collection or array of CacheItemInterface[] items
```

Comment:
@xabbuh thanks for your feedback. The intent here is to say it's an array of CacheItemIterface elements, or a Traverable collection of CacheItermInterface items. As per the function above getItems(). Do you want to suggest an alternative description?